game/scripts/vscripts/components/player/connection.lua

```diff
@@ -5,6 +5,9 @@ if PlayerConnection == nil then
 end
 
 function PlayerConnection:Init()
+  GameEvents:OnPlayerDisconnect(function(keys)
+    PauseGame(true)
```

Comment:
Wouldn't having auto-pause active in all modes, all the time annoy people?

Reply:
Probably, yes. I should check if it's CM.